src/Microsoft.Restier.Core/ApiBase.cs

```diff
@@ -67,7 +67,14 @@ protected ApiConfiguration ApiConfiguration
                         var builder = this.ConfigureApi(new ApiBuilder());
```

Comment:
That's from Autofac, https://github.com/autofac/Autofac. And I also think it acts as the ServiceCollection in MS.DI
And actually I have another topic want to discuss with you.
Initially I wanted to expose IServiceCollection directly for service registration, and then for some reasons added a wrapper class ApiBuilder.
- AddContributor/ChainPrevious APIs, they're somehow unique to RESTier. But actually we could also let them to be extension methods of IServiceCollection.
- Some default services and behaviors in ApiBuilder, but those could also be taken care of with some extension methods.
- Basic service registration could be fulfilled by ApiBuilder, need not a reference to MS.DI, but this only true for simple (and most likely RESTier specific) usage of DI.
- ApiBuilder somehow hides CLSCompliant(false).

Do you prefer IServiceCollection directly or a wrapper like ApiBuilder? Sorry for biinging this up so late, if you like the other way I'd like to take the tedious work. It is said that "Those who pollute should clean up." :)